### PR TITLE
Fix for an unconvential usage of Mocket

### PR DESCRIPTION
**Pull request by mindflayer**
See #191 for details.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ jobs:
         python-version: ["3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy3"]
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
```
Comment (amotl) [inline .github/workflows/main.yml:16]: In order to support Python 3.5 and 3.6 with `actions/setup-python@v4`, you will need to downgrade to `ubuntu-20.04`.

File: mocket/mocket.py [hunk 2]
```diff
@@ -421,6 +421,7 @@ def do_nothing(*args, **kwargs):
 
 
 class Mocket:
+    _address = (None, None)
```
Comment (amotl) [inline mocket/mocket.py:424]: 👍

**Review by amotl — COMMENTED** [review]
Dear Giorgio,

thank you very much for this patch. It looks like it will resolve the problem reported at #191 and https://github.com/hiveeyes/terkin-datalogger/issues/117.

On this matter, I wanted to let you know that one unfortunately can not install the package directly from its Git repository using this command:
```
pip install "git+https://github.com/mindflayer/python-mocket@fix/issue-191"
```

It croaks like:
```python
FileNotFoundError: [Errno 2] No such file or directory: '/private/var/folders/06/w9pzygdj7vx53n_0l9q_lhph0000gn/T/pip-req-build-9h5ttlf8/requirements.txt'
```

No problem so far, I will clone the repository and install the package manually. I just wanted to mention it that it might get a chance to be improved in the future.

With kind regards,
Andreas.

pip install "git+https://github.com/mindflayer/python-mocket@fix/issue-191"

**Comment by amotl** [comment]
> I will clone the repository and install the package manually.

Those commands had to be invoked within the repository tree,
```
pip install pipfile
python -c "import pipfile; pf = pipfile.load('Pipfile'); print('\n'.join(package+version for package, version in pf.data['default'].items()))" > requirements.txt
```

... in order to make the package pip-installable from another location:
```
pip install /path/to/python-mocket
```

After installing it, [Terkin](https://github.com/hiveeyes/terkin-datalogger)'s [LoRaWAN test cases](https://github.com/hiveeyes/terkin-datalogger/blob/0.14.0/test/test_lorawan.py), where we apparently used Mocket unconventionally, succeed again. Thank you so much!

With kind regards,
Andreas.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -8,12 +8,12 @@ jobs:
     runs-on: ubuntu-latest
```
Comment (amotl) [inline .github/workflows/main.yml:8]: See https://github.com/mindflayer/python-mocket/pull/192#discussion_r1038871455.

**Comment by mindflayer** [comment]
I am making Mocket's CI great again, give me a moment and I'll release a new version.

**Comment by mindflayer** [comment]
I had to do [this](https://github.com/actions/setup-python/issues/544#issuecomment-1320295576) to make it work again.

File: .github/workflows/main.yml (hunk 1 above)
Comment (mindflayer) [reply .github/workflows/main.yml:16]: It was `v2` when it first failed.

**Pull request merged**